### PR TITLE
Added logo & icons

### LINKED ISSUE #15: New logo / icon
It would be very nice if we had a logo and a beautiful extension icon. I'm definitely not good with drawing and coloring, so _help wanted_. lol
This is not supposed to be a syntax highlight project only, so the current icon is not the most appropriate.

### PR DESCRIPTION
**Pull request by clarkd**
As discussed, here's the new logos & icons! I've added various icon sizes and have updated the manifest.

I've also updated the README to include the new logo!

Cheers!

Closes #15

### CODE REVIEW

File: logo-full.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 479.02 117.54"><defs><style>.cls-1{font-size:48px;fill:#2684ff;}.cls-1,.cls-2{font-family:Circular-Bold, Circular;font-weight:700;}.cls-2{font-size:69px;}.cls-2,.cls-5{fill:#253858;}.cls-3{letter-spacing:0.01em;}.cls-4{letter-spacing:0em;}</style></defs><title>logo-full</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><text class="cls-1" transform="translate(100.91 41.02)">REFINED</text><text class="cls-2" transform="translate(98.91 100.02)">BITBUCK<tspan class="cls-3" x="296.42" y="0">E</tspan><tspan class="cls-4" x="337.4" y="0">T</tspan></text><path class="cls-5" d="M0,5.52,16.34,100H68.55L86.8,5.52ZM58.46,85H27.19l-9.78-64.5h52Z"/></g></g></svg>
```
Comment (reyronald) [inline logo-full.svg]: I noticed though that the typography in the SVG doesn't look like the original image you posted, can you take a look at that?

**Original** 
original

**This one**
svg

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <sup>-</sup>
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-# Refined Bitbucket
+![Refined BitBucket](./logo-full.svg)
```
Comment (reyronald) [inline README.md:7]: GitHub won't be able to recognize this relative path, it has to be a URL pointing to the resource, use https://rawgit.com/ to get a production URL by giving it the location of the raw file of your latest commit, for example: https://github.com/clarkd/refined-bitbucket/blob/a5c4a2e9859f4e6048308d3ab8ee0ef2ad805c1b/logo-full.svg.

We would end up with something like this:

```markdown
![Refined BitBucket](https://cdn.rawgit.com/clarkd/refined-bitbucket/a5c4a2e9859f4e6048308d3ab8ee0ef2ad805c1b/logo-full.svg)
```

**Review by reyronald — CHANGES_REQUESTED** [review]
Hey @clarkd , again, thank you very much for taking the time to work on this !

I left a few comments for you to look at, let me know when they're ready to be reviewed again!

**Comment by clarkd** [comment]
Hey @reyronald. I think I must have been asleep when I put the PR together, I'll get it fixed up when I get a chance!

File: README.md (hunk 2 above)
Comment (clarkd) [reply README.md:7]: Notably, they do support relative URLs like this - take a look here for example https://github.com/clarkd/refined-bitbucket/tree/work/dc/logos

Comment (reyronald) [reply README.md:7]: **My bad, you are correct**. Maybe it was fixed recently, but that was definitely an issue at some point I think (unless I was asleep when I first stumbled upon it).

However it still persists though in the pull request preview, which led to my confusion:

image

File: logo-full.svg (hunk 1 above)
Comment (clarkd) [reply logo-full.svg]: Oops - I forgot to convert the text to outlines!

File: README.md (hunk 2 above)
Comment (clarkd) [reply README.md:7]: No worries - took me a few tries to get it working and lots of differing opinions on the best way to do it online!

**Comment by clarkd** [comment]
Hey @reyronald, I've just fixed those SVGs so should be good to go now!

**Review by reyronald — APPROVED** [review]

**Comment by reyronald** [comment]
@clarkd Great!! I'm extremely happy with the result! Thank you so much, you will never be forgotten! 😄 

You are awesome!

**Comment by andremw** [comment]
@reyronald I still can't see the new logo in the Chrome Webstore, but in Firefox Store (or whatever the name is) it's fine. Did you have any trouble updating it??

**Comment by reyronald** [comment]
@andremw I assumed Google/Chrome would be smart enough to update it on its own based on the manifest (just like Firefox did), but apparently not... I had to manually go to the dashboard and upload it there by hand just now, like it was 1998 😑 

Check now and let me know if you see it!

**Comment by andremw** [comment]
Now it's fine! @reyronald

**Comment by reyronald** [comment]
Great! Thanks for pointing it out!

**Pull request merged**